.github/workflows/yem-trigger-monitoring.yaml

```diff
@@ -6,34 +6,48 @@ name: yem-trigger-monitoring
 on:
   workflow_dispatch:
   schedule:
-    - cron: '25 12 * * *'
+    - cron: '10 09 * * *' # chirps-gefs should be updated 9AM UTC, let's give an extra 10 min
         
 jobs:
   monitor:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up R 4.2.3
-        uses: r-lib/actions/setup-r@f57f1301a053485946083d7a45022b278929a78a
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
         with:
-          r-version: 4.2.3
-      - name: install required packages
+          r-version: '4.x'
+        # https://github.com/r-spatial/sf/blob/main/.github/workflows/tic-db.yml
+      - name: Install Spatial Libs for Linux
         run: |
-          source(file.path("src", "install_req_r_packages.R"))
-        shell: Rscript {0}
+          sudo apt-get install libgdal-dev libproj-dev libgeos-dev libudunits2-dev
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y libxml2-dev # Add any additional dependencies here
```

Comment:
I would combine the `apt-get installs`, and also start with an `apt-get update`, like [here](https://github.com/OCHA-DAP/Typhoon-Impact-based-forecasting-model/blob/master/.github/workflows/run-typhoon-model.yml#L48)

Reply:
Do you not need [this](https://github.com/OCHA-DAP/Typhoon-Impact-based-forecasting-model/blob/master/.github/workflows/run-typhoon-model.yml#LL47C1-L47C1) PPA for GDAL etc?

Reply:
> I would combine the `apt-get installs`, and also start with an `apt-get update`, like [here](https://github.com/OCHA-DAP/Typhoon-Impact-based-forecasting-model/blob/master/.github/workflows/run-typhoon-model.yml#L48)

good idea - did it here:  dc96d67d7a19123560195f7da57e35832d4b0ae4

> Do you not need [this](https://github.com/OCHA-DAP/Typhoon-Impact-based-forecasting-model/blob/master/.github/workflows/run-typhoon-model.yml#LL47C1-L47C1) PPA for GDAL etc?

apparently not it seems to be running without it